src/main/java/org/folio/des/config/ServiceConfiguration.java

```diff
@@ -149,7 +149,7 @@ EdifactOrdersExportJobScheduler edifactOrdersExportJobScheduler(ScheduledTaskBui
 
   @Bean
   AcqSchedulingProperties acqSchedulingProperties(
-                    @Value("${folio.schedule.acquisition.runOnlyIfModuleRegistered:true}") String runOnlyIfModuleRegistered) {
+                    @Value("${folio.schedule.acquisition.runOnlyIfModuleRegistered:false}") String runOnlyIfModuleRegistered) {
```

Comment:
This is just for debugging purposes, should be removed before the commit

Reply:
Fixed.